README.md

```diff
@@ -52,14 +52,20 @@ $layer->class->add($class);
 $map->layer->add($layer);
 
 $mapfile = (new \MapFile\Writer\Map())->write($map);
+
+// or
+
+$writer = new \MapFile\Writer\Map('my-mapfile.map');
+$writer->write($map);
+$writer->save();
```

Comment:
I agree with you here but not related to OUTPUTFORMAT.
This will be tackled in a next PR.
Please revert.